app/assets/stylesheets/application.scss

```diff
@@ -15,22 +15,266 @@
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Ferra-Torr {
+    background-image: image-url("character_backgrounds/Ferra-Torr-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Goro {
+    background-image: image-url("character_backgrounds/Goro-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Jacqui-Briggs {
+    background-image: image-url("character_backgrounds/Jacqui-Briggs-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Jax {
+    background-image: image-url("character_backgrounds/Jax-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Johnny-Cage {
+    background-image: image-url("character_backgrounds/Johnny-Cage-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kano {
+    background-image: image-url("character_backgrounds/Kano-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kenshi {
+    background-image: image-url("character_backgrounds/Kenshi-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kitana{
+    background-image: image-url("character_backgrounds/Kitana-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kotal-Kahn {
+    background-image: image-url("character_backgrounds/Kotal-Kahn-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kung-Jin {
+    background-image: image-url("character_backgrounds/Kung-Jin-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kung-Lao {
+    background-image: image-url("character_backgrounds/Kung-Lao-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Liu-Kang {
+    background-image: image-url("character_backgrounds/Liu-Kang-Left.png");
```

Comment:
Properties should be ordered background-attachment, background-color, background-image, background-image, background-size, position